fronts-client/src/components/feed/RecipeSearchContainer.tsx

```diff
@@ -38,34 +39,53 @@ const FeastSearchContainerComponent = ({
   rightHandContainer,
   recipes,
   chefs,
-}: Props) => (
-  <React.Fragment>
-    <InputContainer>
-      <TextInputContainer>
-        <TextInput placeholder="Search recipes" displaySearchIcon />
-      </TextInputContainer>
-      <ClipboardHeader />
-    </InputContainer>
-    <FixedContentContainer>
-      <SearchResultsHeadingContainer>
-        <SearchTitle>
-          {'Results'}
-          <ShortVerticalPinline />
-        </SearchTitle>
-      </SearchResultsHeadingContainer>
-      {Object.values(recipes).map((recipe) => (
-        <RecipeFeedItem key={recipe.id} recipe={recipe} />
-      ))}
-      {Object.values(chefs).map((chef) => (
-        <ChefFeedItem
-          key={chef.id}
-          chef={chef}
-        /> /*TODO: as of now, added rendering of chefs just after the recipes. Need
-      to change when "search-chefs" action gets finalised*/
-      ))}
-    </FixedContentContainer>
-  </React.Fragment>
-);
+}: Props) => {
+  const [selectedOption, setSelectedOption] = useState<string>('recipeFeed');
+  const handleOptionChange = (optionName: string) => {
+    setSelectedOption(optionName);
+  };
+  return (
+    <React.Fragment>
+      <InputContainer>
+        <TextInputContainer>
+          <TextInput placeholder="Search recipes" displaySearchIcon />
+        </TextInputContainer>
+        <ClipboardHeader />
+      </InputContainer>
+      <RadioGroup>
+        <RadioButton
+          checked={selectedOption === 'recipeFeed'}
+          onChange={() => handleOptionChange('recipeFeed')}
+          label="Recipes"
+          inline
+          name="recipeFeed"
+        />
+        <RadioButton
+          checked={selectedOption === 'chefFeed'}
+          onChange={() => handleOptionChange('chefFeed')}
+          label="Chefs"
+          inline
+          name="chefFeed"
+        />
+      </RadioGroup>
+      <FixedContentContainer>
+        <SearchResultsHeadingContainer>
+          <SearchTitle>
+            {'Results'}
+            <ShortVerticalPinline />
+          </SearchTitle>
+        </SearchResultsHeadingContainer>
+        {selectedOption === 'recipeFeed'
+          ? Object.values(recipes).map((recipe) => (
+              <RecipeFeedItem key={recipe.id} recipe={recipe} />
+            ))
+          : Object.values(chefs).map((chef) => (
+              <ChefFeedItem key={chef.id} chef={chef} />
+            ))}
+      </FixedContentContainer>
```

Comment:
I think that this is fine for the time being, but the ternary syntax doesn't extend very nicely if we want to put a third option in.  To be honest my preference here would be to use a second functional component in the file which contains a `switch` block on the `selectedOption` value (passed over as a param) but I wouldn't bother making that change now unless we know that we _are_ going to need a third case :grin:

Reply:
Yeah changed to switch case, thanks. This will help in scalability.